From: Sven Eckelmann <sven@narfation.org>
Date: Thu, 16 Jun 2016 09:40:19 +0200
Subject: FTBFS AM_C_PROTOTYPES support

Build fails because of "error: automatic de-ANSI-fication support has been
removed" during the ./autogen.sh call when aclocal-1.15 and automake-1.16
is used. But the control file expects automake/aclocal-1.11 to be
installed. So enforce the call to these version of tools instead of the
versions selected as default by the system.

Bug-Debian: https://bugs.debian.org/822544
---
 autogen.sh | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/autogen.sh b/autogen.sh
index e0d6691..8eed97b 100644
--- a/autogen.sh
+++ b/autogen.sh
@@ -3,7 +3,7 @@
 # invoke the auto* tools to create the configuration/build system
 
 # build aclocal.m4
-aclocal
+aclocal-1.11
 
 # build config.h
 autoheader
@@ -15,7 +15,7 @@ autoconf
 libtoolize --force
 
 # invoke automake
-automake --foreign --add-missing --ignore-deps
+automake-1.11 --foreign --add-missing --ignore-deps
 
 # and finally invoke our new configure
 [ -n "$NOCONFIGURE" ] || ./configure $*
